scripts/setup-ubuntu.sh

```diff
@@ -84,9 +83,21 @@ function install_folly {
   cmake_install -DBUILD_TESTS=OFF
 }
 
+function install_protobuf {
+  wget https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERS}/protobuf-all-${PROTOBUF_VERS}.tar.gz
+  tar -xzf protobuf-all-${PROTOBUF_VERS}.tar.gz -C ${PROTOBUF_INSTALL_DIR}
+  cd ${PROTOBUF_INSTALL_DIR}/protobuf-${PROTOBUF_VERS}/ 
+  ./configure && make "-j${NPROC}" && make install
+  ldconfig
+  ln /usr/local/bin/protoc /usr/bin/protoc
+  VERSION=`protoc --version`
+  echo ${VERSION}
+}
+
 function install_velox_deps {
   run_and_time install_fmt
   run_and_time install_folly
+  run_and_time install_protobuf
```

Comment:
Should we update other setup-xxx.sh scripts?

@kgpai @majetideepak Krishna, Deepak, would you help review the setup script changes?

Reply:
> Should we update other setup-xxx.sh scripts?
> 
> @kgpai @majetideepak Krishna, Deepak, would you help review the setup script changes?

And please also check if we need do some changes in setup-circleci.sh. because i find "dnf_install protobuf-devel" there,